lib/Logger.jsx

```diff
@@ -81,60 +73,51 @@ export default class Logger {
      * @param {String} message The message to log.
      * @param {Boolean} sendNow if true, the message will be sent right away.
      * @param {Object|String} parameters The parameters to send along with the message
+     * @param {Boolean} onlyFlushWithOthers A request will never be sent to the server if all loglines have this set to true
```

Comment:
I added this param for 2 reasons:
1. I did not want it to be sending a request to the server with only [this message](https://github.com/Expensify/expensify-common/pull/397/files#diff-8d2a2ad4daaacb78979ad7736962c1225e97d46921f1cefd010c7a9f68556af8R39) as that would be useless
2. Same thing happens with [this log line](https://github.com/Expensify/App/pull/4191/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R38)